.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": false,
+    "jsxSingleQuote": false,
+    "bracketSpacing": true
+}
```

Comment:
Maybe you can remove some settings which is same as default value to keep it simple